README.md

```diff
@@ -1,13 +1,11 @@
 # Project Happy Thoughts API
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+This week projects is to practise creating endpoints using POST methods besides reinforcing the skills of using GET method with moongoose library.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+It is very interesting to see the connection between front end, backend and database in this project. I did not have major problems.
 
 ## View it live
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+https://whatmakesyouhappy.herokuapp.com/thoughts
```

Comment:
It would have been nice to see your frontend using this API to see that they work together :-)

You could have updated you frontend code using this API and added the URL link here.